### DISCUSSION TITLE
Unable to initialize evennia on new laptop

### DISCUSSION THREAD

**Discussion by pruuk** [evennia/evennia#3631] [Evennia Questions]
I'm trying to start from a fresh copy of Evennia on a M1 mac airbook running 14.6.1. Steps I've taken:
1. installed python 3.12.7 (latest)
2. Created a virtual env running python 3.12.7 (vs. machine default of 3.9)
3. Cloned the evennia repo from git
4. tried running evennia --init <dir name>

Nothing happens at this point. I get a message:
"(zsh: command not found: evennia)

I've also tried running the command from within the evennia folder and the evnnis folder within that one. Any ideas?

**Comment by pruuk** [comment]
Solved: I was able to get an install working by installing via pip instead of cloning the repo with git.

**Reply by InspectorCaracal** [reply]
Ah you got it while I was typing xD Good luck with your project!

**Comment by InspectorCaracal** [comment]
You don't need to clone the repo unless you specifically want to keep up with cutting-edge development commits.

In both cases, though, the step you missed was actually installing evennia.

While your virtual environment is active, use one of the following:

*Installing the latest official release from pypi* (recommended)
```
pip install evennia
```

*Installing from the cloned git repository (must be in the same directory as your evennia directory, NOT within the evennia directory)*
```
pip install -e evennia
```

After installing evennia, you will be able to use the evennia commands like `evennia --init`
